sbt/src/sbt-test/source-dependencies/macro-auxiliary-files/test

```diff
@@ -0,0 +1,10 @@
+> compile
+
+# Modify the auxiliary file
+$ copy-file changes/util.txt util/util.txt
+
+$ copy-file changes/Bar.scala macro-provider/src/main/scala/Bar.scala
+
+> macro-client/compile
+
+> macro-client/checkRecompiled Client.scala
```

Comment:
Hello Greg! It's defined here: https://github.com/sbt/sbt/pull/1757/files#diff-2097d5b5a7f0292a54187e2fb573c989R2, in  `sbt/src/sbt-test/source-dependencies/macro-auxiliary-files/macro-client/build.sbt`